### PR TITLE
Fix #3640: Make 'remove from playlist' button destructive.

### LINKED ISSUE #3640: Make 'remove from playlist' button destructive.
Zrzut ekranu 2021-05-6 o 17 23 55

### PR DESCRIPTION
**Pull request by steven851007**
## Summary of Changes

This pull request fixes  #3640

## Submitter Checklist:

- [x] *Unit Tests* are updated to cover new or changed functionality
- [x] User-facing strings use `NSLocalizableString()`

## Screenshots:
Screenshot 2021-05-12 at 09 50 35

## Reviewer Checklist:

- [x] Issues include necessary QA labels:
  - `QA/(Yes|No)`
  - `release-notes/(include|exclude)`
  - `bug` / `enhancement`
- [x] Necessary [security reviews](https://github.com/brave/security/issues/new/choose) have taken place.
- [x] Adequate unit test coverage exists to prevent regressions.
- [x] Adequate test plan exists for QA to validate (if applicable).
- [x] Issue is assigned to a milestone (should happen at merge time).

### CODE REVIEW

**Review by iccub — APPROVED** [review]
Looking good, thanks :)

**Comment by steven851007** [comment]
> Looking good, thanks :)

Thanks, my first PR in the project I'll try to contribute more in future too :)
But I can't merge the MR as I don't have write access, feel free to merge it

**Comment by iccub** [comment]
That's great to hear @steven851007, you can let us know when you want to pickup a ticket, we can then guide you or tell you if someone at Brave team already working on it

**Pull request merged**
